xml/System.Runtime.Caching.Configuration/CachingSectionGroup.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Defines a configuration section for.NET Framework caching. This class cannot be inherited.</summary>
+    <summary>Defines a configuration section for .NET Framework caching. This class cannot be inherited.</summary>
```

Comment:
Shouldn't this one be just .NET too?
```suggestion
    <summary>Defines a configuration section for .NET caching. This class cannot be inherited.</summary>
```

Reply:
I left it as .NET Framework because line 10 (assembly version) says this only applies to .NET Framework. Is that correct?